docs/content/en/blog/releases/v0.35.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Release v0.35.0"
+linkTitle: "Release v0.35.0"
+date: 2022-08-04
+description: >
+ Release v0.35.0
+---
+
+## Changes since v0.34.1
+
+> Starting from version v0.35.0, the CloudProvider concept is being replaced by PlatformProvider. It's a name change due to the PipeCD vision improvement. __The CloudProviders configuration is marked as deprecated, please migrate your piped agent configuration to use PlatformProviders__.
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Piped
+spec:
+  ...
+  platformProviders:
+    - name: kubernetes-dev
+      type: KUBERNETES
+```
+
+Checkout the docs in case you do not know what to do. ref: [docs](http://pipecd.dev/docs/operator-manual/piped/adding-a-platform-provider/)
+
+### Notable Changes
+
+* Update k8s executor to follow specified routes when applying manifests ([#3817](https://github.com/pipe-cd/pipecd/pull/3817))
```

Comment:
This can be moved to the "Internal Changes" section since the `multi-provider` feature is not published yet.

Reply:
Sure, addressed by https://github.com/pipe-cd/pipecd/pull/3833/commits/0e73fab2836ab808e98982352b07f16d8f401741 🙏